meilisearch/client.py

```diff
@@ -280,25 +280,24 @@ def create_key(
         MeiliSearchApiError
             An error containing details about why MeiliSearch can't process your request. MeiliSearch error codes are described here: https://docs.meilisearch.com/errors/#meilisearch-errors
         """
-        if options is None:
-            options= {}
         payload = {**options}
```

Comment:
`payload = {**options}` will unpack the `options` dict into a new identical dict called `payload`. Since nothing is being changed you can instead directly send `options` and the result will be the same with one less step.

```py
def create_key(
    self,
    options: Dict[str, Any],
) -> Dict[str, int]:
    return self.http.post(f'{self.config.paths.keys}', options)
```